Packs/DigitalGuardian/README.md

```diff
@@ -0,0 +1,19 @@
+# Digital Guardian
+
+This pack integrates with Digital Guardian's Data Loss Prevention (DLP) platform Analytics & Reporting Cloud (ARC) solution to identify, remediate, and protect sensitive data from insider and outsider threats.
+
+## What does this pack do?
+
+* Automates alert fetching from user-configured export profiles.
+
+* Allows for the execution of response actions to mitigate risks without manual intervention.
+
+Customers can decide which events and alerts are integrated by selecting a Digital Guardian export profile when configuring the event collector, ensuring visibility is bespoke to your individual SOC’s needs.
+
+<~XSIAM>
+Records fetched by the event collector the event collector will appear in the XSIAM ‘Alerts’ page.
+</~XSIAM>
+
+<~XSOAR>
+Records fetched by the event collector the event collector will appear in the XSOAR ‘Incidents’ page.
+</~XSOAR>
```

Comment:
I think we have an automated process for that - let's discuss offline

Reply:
Removed the `<~XSOAR> ... </~XSOAR>` part since the event collector is designed for XSIAM

Please review the `README.md` changes in commit ef814676d12876d5adc3cde92e5f7ff92c595e24

Reply:
If the pack is only for XSIAM, there's no need for the `<XSIAM>` tag either.